src/gui/CemuApp.cpp

```diff
@@ -16,6 +16,11 @@
 #include "gui/helpers/wxWayland.h"
 #endif
 
+//GameMode support
+#if BOOST_OS_LINUX && ENABLE_FERAL_GAMEMODE
```

Comment:
```suggestion
#if BOOST_OS_LINUX && defined(ENABLE_FERAL_GAMEMODE)
```
Imho it's more robust to check if `ENABLE_FERAL_GAMEMODE` is defined instead of checking if the value is non-zero since it's not assigned an explicit value.